doc/website/advanced/iceoryx_utils.md

```diff
@@ -1,4 +1,4 @@
 ---
 title: Safe building blocks
 ---
-{! ./../iceoryx_utils/README.md !}
+{! ../iceoryx/iceoryx_utils/README.md !}
```

Comment:
How can a reviewer ensure that the relative links are correct now? Where can I see where the starting point for this relative link is?

Reply:
By running either `mkdocs serve` or `mike deploy`, see [export script](https://github.com/eclipse-iceoryx/iceoryx/blob/master/tools/export-docu-to-website.sh#L74-L86).

If no [`base_path`](https://github.com/cmacmackin/markdown-include/#configuration) is specified when using `markdown-include`...

> Default: the run-directory

Reply:
I've added a sanity check in the export script.